### PR TITLE
nextjs deployment

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: applications/next-js/deployment/dev/application.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        image: <image>
+        image_name: next-js-app
+        port: 8080
+      traits:
+        - type: path-based-ingress
+          properties:
+            domain: "*.elb.us-west-2.amazonaws.com"
+            rewritePath: true
+            http:
+              /unicorn: 8080
```
Comment (shapirov103) [inline applications/next-js/deployment/dev/application.yml:19]: accessible on /unicorn

File: applications/next-js/lib/dynamo/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { v4 as uuidv4 } from 'uuid';
 import { Menu, Page, Product, Category, Cart, CartProduct, ProductVariants } from './types';
 
-const API_URL = process.env.API_BASE_URL ?? 'http://rust-backend';
+const API_URL = process.env.API_BASE_URL ?? 'http://rust-backend.team-rust,svc.cluster.local';
```
Comment (5herlocked) [inline applications/next-js/lib/dynamo/index.ts]: errant `,`

**Review by 5herlocked — CHANGES_REQUESTED** [review]

File: applications/next-js/lib/dynamo/index.ts (hunk 2 above)
Comment (elamaran11) [reply applications/next-js/lib/dynamo/index.ts]: +1

Comment (shapirov103) [reply applications/next-js/lib/dynamo/index.ts]: fixed

**Review by elamaran11 — APPROVED** [review]
LGTM

**Comment by elamaran11** [comment]
@5herlocked You have a changes requested. Please approve, i can merge

**Review by 5herlocked — APPROVED** [review]

**Pull request merged**